flixel/system/macros/FlxDefines.hx

```diff
@@ -254,7 +254,9 @@ class FlxDefines
 			define(FLX_TRACK_GRAPHICS);
 
 		#if (lime_opengl || lime_opengles || lime_webgl)
-		define(FLX_OPENGL_AVAILABLE);
+		// FlxG.stage.window.context.attributes.hardware is not always defined during unit tests
```

Comment:
Would it be better to check if `FlxG.stage.window.context.attributes.hardware` (or whatever part of it that fails) is defined rather than outright skipping it?

Reply:
I don't think so. If it's not defined in any other context I want it to fail loudly, and I don't think we can really unit test gl stuff right now